project/project.go

```diff
@@ -238,21 +265,36 @@ func (p *Project) load(file string, bytes []byte) error {
 			}
 		}
 	}
+}
 
-	// FIXME(vdemeester) Not sure about this..
-	if p.context.NetworksFactory != nil {
-		networks, err := p.context.NetworksFactory.Create(p.Name, p.NetworkConfigs, p.ServiceConfigs, p.isNetworkEnabled())
-		if err != nil {
-			return err
-		}
+func (p *Project) isNetworkEnabled() bool {
+	return p.configVersion == "2"
+}
 
-		p.networks = networks
+func (p *Project) handleVolumeConfig() {
```

Comment:
Not a huge deal, but can some of the nesting be removed in this function? It'd be a little easier to read.